oracledb-ballerina/tests/03_connection_pool_test.bal

```diff
@@ -0,0 +1,583 @@
+//   resultArray[1] = oracleDbClient1->query("select count(*) as val from Customers where registrationID = 2", Result);
+//
+//   resultArray[2] = oracleDbClient2->query("select count(*) as val from Customers where registrationID = 1", Result);
+//   resultArray[3] = oracleDbClient2->query("select count(*) as val from Customers where registrationID = 1", Result);
+//
+//   resultArray[4] = oracleDbClient3->query("select count(*) as val from Customers where registrationID = 2", Result);
+//   resultArray[5] = oracleDbClient3->query("select count(*) as val from Customers where registrationID = 2", Result);
+//
+//   resultArray[6] = oracleDbClient4->query("select count(*) as val from Customers where registrationID = 1", Result);
+//   resultArray[7] = oracleDbClient4->query("select count(*) as val from Customers where registrationID = 1", Result);
+//
+//   resultArray[8] = oracleDbClient5->query("select count(*) as val from Customers where registrationID = 1", Result);
+//   resultArray[9] = oracleDbClient5->query("select count(*) as val from Customers where registrationID = 1", Result);
+//
+//   resultArray[10] = oracleDbClient5->query("select count(*) as val from Customers where registrationID = 1", Result);
+//
+//   (int|error)[] returnArray = [];
+//   int i = 0;
+//   // Connections will be released here as we fully consume the data in the following conversion function calls
+//   foreach var x in resultArray {
+//       returnArray[i] = getReturnValue(x);
+//       i += 1;
+//   }
+//   // All 5 clients are supposed to use the same pool. Default maximum no of connections is 10.
+//   // Since each select operation hold up one connection each, the last select operation should
+//   // return an error
+//   i = 0;
+//   while(i < 10) {
+//       test:assertEquals(returnArray[i], 1);
+//       i = i + 1;
+//   }
+//   validateConnectionTimeoutError(returnArray[10]);
+//}
+//
+//function getReturnValue(stream<record{}, error> queryResult) returns int|error {
+//   int count = -1;
+//   record {|record {} value;|}? data = check queryResult.next();
+//   if (data is record {|record {} value;|}) {
+//       record {} value = data.value;
+//       if (value is Result) {
+//           count = value.val;
+//       }
+//   }
+//   check queryResult.close();
+//   return count;
+//}
+//
+//function validateApplicationError(int|error dbError) {
+//   test:assertTrue(dbError is error);
+//   sql:ApplicationError sqlError = <sql:ApplicationError> dbError;
+//   test:assertTrue(stringutils:includes(sqlError.message(), "Client is already closed"), sqlError.message());
+//}
+//
+//function validateConnectionTimeoutError(int|error dbError) {
+//   test:assertTrue(dbError is error);
+//   sql:DatabaseError sqlError = <sql:DatabaseError> dbError;
+//   test:assertTrue(stringutils:includes(sqlError.message(), "request timed out after"), sqlError.message());
+//}
```

Comment:
Any reason to comment these test cases?. if not, shall we enable them.

Reply:
Added with b009a6293fdf5d442706439124fb9e8d297d076c